### DISCUSSION TITLE
冷门需求，增加自动切换原生幻灯片壁纸功能

### DISCUSSION THREAD

**Discussion by peterwaye** [Lunova-Studio/DarkMode2#6] [Show and tell]
收集了一套亮色、一套暗色的幻灯片壁纸，希望能在切换主题时能跟着切换。

目前想到的一种实现方式是：
创建group1、group2两个文件夹。
win10个性化设置group1为幻灯片文件夹。
切换时，
如果存在group1、group2，把group1改名为ref，group2改名为group1；
如果存在group1、ref，把group1改名为group2，ref改名为group1。

**Comment by bao-cn** [comment]
可开启社区投票，如果一个月内投票支持率达到80%，可追加新功能项目计划
